NuRadioMC/examples/05_pulser_calibration_measurement/radioEmitting_pulser_calibration/config.yaml

```diff
@@ -0,0 +1,15 @@
+noise: False  # specify if simulation should be run with or without noise
+sampling_rate: 2.  # sampling rate in GHz used internally in the simulation.
+speedup:
+  minimum_weight_cut: 1.e-5
```

Comment:
I think you need to deactivate the minimum_weight_cut, i.e. `minimum_weight_cut: 0`. I think for your case it has not effect because the zenith angle is `0`, but better deactivate this feature.